pyzx/circuit/gates.py

```diff
@@ -1256,6 +1258,24 @@ def to_graph(self, g, q_mapper, c_mapper):
             g.add_edge((v,u), EdgeType.SIMPLE)
             c_mapper.set_next_row(self.result_bit, r+1)
 
+    def to_graph_symbolic_boolean(self, g, q_mapper):
+        """Represent the measurement as a node with symbolic boolean phases."""
+        r = q_mapper.next_row(self.target)
+        phase = new_var(name=f"a{self.target}", is_bool=True, registry=g.var_registry) 
+        _ = self.graph_add_node(g,
+            q_mapper,
+            VertexType.X,
```

Comment:
I just wanted to double-check that this is correct/intended. Isn't the default basis for measurement in OpenQASM the Z-basis? Or did I miss an implicit Hadamard somewhere?